source/rust_verify/example/state_machines/top_sort_dfs.rs

```diff
@@ -43,7 +43,7 @@ impl<V> DirectedGraph<V> {
 }
 
 tokenized_state_machine!{
-    TopSort<#[verifier(maybe_negative)] V> {
+    TopSort<#[verus::verifier(maybe_negative)] V> {
```

Comment:
@tjhance it looks like this isn't being replaced by the `verus!` macro, which now will replace `verifier` with `verus::verifier`; my understanding is that the state machine macros run (parts of) the token stream through the `verus!` macro, is that correct?

Reply:
that's right, but only part of it. i think there are a few places like that cause awkwardness and need to be fixed